src/components/Map/Map.tsx

```diff
@@ -458,7 +459,12 @@ export default function Map(props: MapProps): JSX.Element {
             destroying = true;
           }}
           ref={mapRefCb}
-          onRender={(event) => event.target.resize()}
+          onRender={(event) => {
+            if (!resized) {
```

Comment:
This is the fix. @charlie-tf if this somehow breaks what you had originally intended we can follow up.

Reply:
Just checked it out, and it appears to still work as expected 👍

Reply:
Great, thanks!!